<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraDisplay</class>
 <widget class="QDialog" name="CameraDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="effect_box">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="camera_box">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="record_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="picture_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
  </widget>
  <widget class="QSlider" name="zoom_slider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>961</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layout"/>
  </widget>
  <widget class="QPushButton" name="clear_grid_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Grid</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_grid_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggle_path_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Toggle Path</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_path_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
